packages/firestore/src/remote/remote_store.ts

```diff
@@ -924,3 +895,111 @@ export async function remoteStoreApplyPrimaryState(
     remoteStoreImpl.onlineStateTracker.set(OnlineState.Unknown);
   }
 }
+
+/**
+ * Registers a callback function that is invoked when the network status
+ * changes. Multiple callbacks are supported.
+ */
+function addNetworkStatusChangedHandler(
```

Comment:
Should this just be `onNetworkStatusChange` to match other event handlers we have?

Reply:
I inlined this and changed the internal variable name to `onNetworkStatusChange`.